src/app/watchlist/handler/createWatchlistHandler.go

```diff
@@ -41,5 +41,5 @@ func (controller *CreateWatchlistController) HandleCreateWatchlist(ctx *gin.Cont
 		ctx.Abort()
 		return
 	}
-	ctx.JSON(http.StatusCreated, gin.H{genericConstants.BFFResponseSuccessMessage: serviceConstants.WatchlistCreatedSuccessfully})
+	ctx.JSON(http.StatusOK, gin.H{genericConstants.BFFResponseSuccessMessage: serviceConstants.WatchlistCreatedSuccessMessage})
```

Comment:
The previous code is correct. If you create something, then its error code should be "http.StatusCreated"

Reply:
Resolved